leshan-core/src/main/java/org/eclipse/leshan/core/request/DownlinkRequestVisitor2.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Sierra Wireless and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Sierra Wireless - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.leshan.core.request;
+
+/**
+ * A new version of {@link DownlinkRequestVisitor} which supports {@link BootstrapDiscoverRequest}
+ * 
+ * @since 1.1
+ */
+public interface DownlinkRequestVisitor2 extends DownlinkRequestVisitor {
```

Comment:
Another solution could be to update the existing interface and provide a default implementation so that we don't need a new one. wdyt?
Same for LwM2mObjectEnabler.

Reply:
(same reason : https://github.com/eclipse/leshan/pull/856#discussion_r451645352)